phpseclib/File/X509.php

```diff
@@ -3794,7 +3794,7 @@ function &_subArray(&$root, $path, $create = false)
      * @param string $path optional absolute path with / as component separator
      * @param bool $create optional
      * @access private
-     * @return array ref or false
```

Comment:
The return value may have a description though, right? So one option would be to describe in the text that the return value is a reference.